src/main/java/core/basesyntax/operationswithfile/FileWriterImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.operationswithfile;
+
+import java.io.IOException;
+import java.io.StringWriter;
+import java.util.HashMap;
+import java.util.Map;
+import org.supercsv.io.CsvListWriter;
+import org.supercsv.io.ICsvListWriter;
+import org.supercsv.prefs.CsvPreference;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void getNewFile(HashMap<String, Integer> balance, String newCsvFileName) {
```

Comment:
In my opinion we should take Map if we want follow SOLID principles

Reply:
add